public/index.html

```diff
@@ -25,6 +25,16 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Breads</title>
+
+    <!-- Below JSDelivr CDN and Widget script for DarkMode -->
+    <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.7/lib/darkmode-js.min.js"></script>
```

Comment:
Add `defer` to the end of the first script tag to make so this script isn't executed until the UI has been loaded